gamestonk_terminal/helper_classes.py

```diff
@@ -2,12 +2,15 @@
 __docformat__ = "numpy"
 import os
 import json
+
 from importlib import machinery, util
 from typing import Union, List, Dict, Optional
 import matplotlib
 import matplotlib.pyplot as plt
 from matplotlib import font_manager
 
+import gamestonk_terminal.feature_flags as gtff
```

Comment:
this will not work when using the notebook api because gtff is not fully initialized at the time helper classes are initialized